src/hooks/jitPluginInstall.ts

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2022, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+import * as os from 'os';
+import { Hook } from '@oclif/core';
+import { SfError } from '@salesforce/core';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-expect-error
+import { TelemetryGlobal } from '@salesforce/plugin-telemetry/lib/telemetryGlobal';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-expect-error
+import { AppInsights } from '@salesforce/telemetry/lib/appInsights';
+
+declare const global: TelemetryGlobal;
+
+const hook: Hook<'jit_plugin_not_installed'> = async function (opts) {
+  try {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-member-access, @typescript-eslint/no-unsafe-call
+    global.cliTelemetry?.record({
+      eventName: 'JIT_INSTALL_STARTED',
+      type: 'EVENT',
+      version: opts.config.version,
+      plugin: opts.command.pluginName,
+      command: opts.command.id,
+    });
+
+    await opts.config.runCommand('plugins:install', [`${opts.command.pluginName}@${opts.pluginVersion}`]);
```

Comment:
note: there's a current bug in the `verify:signature` hook that makes it throw an err if a semver value is specified.

IIRC, it only works when it's a specific version available on npm. Carete'd plugins will not work.

[W-11771849](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE000016feSfYAI/view)